### PR TITLE
Added command line arguments to account for mean/std dev and scale on…

### PR DESCRIPTION
**Pull request by pmbrown1055**
… input images to allow for result validation.  Cleaned up some performance measurement issues (removed 'SUCCESS' print from inner loop).  Added CPU memory measurement to GPU runs for run comparison.

### CODE REVIEW

**Comment by ryanlai2** [comment]
Hey Paul! Would it be possible for you to write some tests for this change? Are you able to run the existing winmlrunner tests in Visual Studio?

File: Tools/WinMLRunner/CommandLineArgs.h [hunk 1]
```diff
@@ -34,6 +37,9 @@ class CommandLineArgs
     std::wstring m_modelFolderPath;
     std::wstring m_modelPath;
     std::wstring m_imagePath;
+
+	float m_scale;
```
Comment (ryanlai2) [inline Tools/WinMLRunner/CommandLineArgs.h]: This scale member variable needs to be initialized. In line 339 in BindingUtilities.h, you check if m_scale!=1.0
```suggestion
	float m_scale = 1.0;
```

**Comment by ryanlai2** [comment]
Hey Paul! Just checking in with you, would you be able to provide some more information on the expected behavior of your image scaling changes? Is there some input parameters I can pass into winmlrunner.exe and look at the expected output? Also, there have been some big changes since your PR so a merge may be needed to update this PR.

Thanks!

**Comment by pmbrown1055** [comment]
There are some more changes we are working on to generalize this change, and general enhancements.  Will also come back with unit tests for the feature that are implemented, and the other initialization fixed that you mentioned.

Thanks!

**Pull request closed without merging**